deeplinkdispatch-processor/src/main/java/com/airbnb/deeplinkdispatch/DeepLinkAnnotatedElement.java

```diff
@@ -26,13 +27,14 @@
   private final TypeElement annotatedElement;
   private final String method;
   private final Element element;
+  private final String[] prefixes;
+
+  DeepLinkAnnotatedElement(String annotation, Element element, DeepLinkEntry.Type type,
+          String[] prefixes) throws MalformedURLException {
+    this.prefixes = prefixes;
+
+    validateAnnotation(annotation);
```

Comment:
It's an anti-pattern to throw from a constructor. Either call this method separately after you construct the object or use a builder. While you're at it, I'd also turn `prefixes` into a `List<String>` just because it's easier to work with